linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_after_time/rule.yml

```diff
@@ -21,6 +21,7 @@ identifiers:
 
 references:
     ospp: FMT_SMF_EXT.1
+    srg: SRG-OS-000029-GPOS-00010
 
 ocil_clause: 'lock-after-time is not set or set to zero'
```

Comment:
This rule appears to need updating. The requirement is to lock after 15minutes.

Reply:
Description is wrong, agreed. OVAL and remediations are hardcoded to 900 at the moment.

Reply:
https://github.com/ComplianceAsCode/content/issues/4965 was opened for this.